bundles/EcommerceFrameworkBundle/CartManager/CartInterface.php

```diff
@@ -40,12 +40,12 @@ interface CartInterface
     const COUNT_MAIN_AND_SUB_ITEMS = 'main_and_sub';
 
     /**
-     * @return int|null
+     * @return string|int|null
```

Comment:
SessionCart use string as id see:
https://github.com/pimcore/pimcore/blob/87972b48f4293cd1b54641ef3c2fc708a63de718/bundles/EcommerceFrameworkBundle/CartManager/SessionCart.php#L61
Cart use int as id because the ecommerceframework_cart db table id column is an integer:
https://github.com/pimcore/pimcore/blob/87972b48f4293cd1b54641ef3c2fc708a63de718/bundles/EcommerceFrameworkBundle/CartManager/Cart/Dao.php#L69

So the interface should allow both at the moment.